core/src/test/scala/unit/kafka/admin/LogDirsCommandTest.scala

```diff
@@ -0,0 +1,52 @@
+package unit.kafka.admin
```

Comment:
We must add the licence header here. You can copy it from another file.

Reply:
Sorry, after checking the compilation report, I have realized this problem and I have made changes.